tests/lib/nested.sh

```diff
@@ -141,48 +148,40 @@ get_ubuntu_image_url_for_nested_vm() {
         esac
 }
 
-get_cdimage_current_image_url() {
+nested_get_image_url_for_vm() {
+    if [[ "$SPREAD_BACKEND" == google* ]]; then
+        #shellcheck disable=SC2119
+        nested_get_google_image_url_for_vm
+    else
+        nested_get_ubuntu_image_url_for_vm
```

Comment:
this is wrong, we used `nested_get_image_url_for_vm` like this:

```
nested_get_image_url_for_vm ubuntu-20.04-64
```

and so this function needs to pass it's arguments through like this:

```suggestion
        nested_get_google_image_url_for_vm "$@"
    else
        nested_get_ubuntu_image_url_for_vm "$@"
```

Reply:
thanks